paddle/operators/gru_op.h

```diff
@@ -102,8 +103,10 @@ class GRUKernel : public framework::OpKernel<T> {
       gru_value.reset_output_value = reset_hidden_prev_t.data<T>();
       math::GRUUnitFunctor<DeviceContext, T>::compute(
           dev_ctx, gru_value, frame_size, cur_batch_size,
-          math::ActiveType(context.Attr<std::string>("activation")),
-          math::ActiveType(context.Attr<std::string>("gate_activation")));
+          math::detail::GetActivationType(
+              context.Attr<std::string>("activation")),
+          math::detail::GetActivationType(
+              context.Attr<std::string>("gate_activation")));
```

Comment:
Put the ` math::detail::GetActivationType` before the for loop in line 93. The same as bellow.

Reply:
Done.